client/my-sites/posts/main.jsx

```diff
@@ -99,6 +100,7 @@ class PostsMain extends React.Component {
 				<div className="posts__primary">
 					<PostTypeFilter query={ query } siteId={ siteId } statusSlug={ statusSlug } />
 					{ siteId && <PostTypeBulkEditBar /> }
+					{ siteId && isJetpack && <QueryJetpackModules siteId={ siteId } /> }
```

Comment:
note that except in rare cases we want to match the query component with the selector getting data. did we have a reason to add this here but not where we depend on the data? it can get confusing particularly on future cleanups why this exists since the dependency is broken across files.

Reply:
Agreed! I expected to see it on the `client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx` file.

Reply:
I think I needed it here to have access to the `siteId` in the first place? But I may have been confusing something, or not understanding how to get it in `client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx`.

Reply:
You can inject a new `siteId` prop in the `mapStateToProps` function:

https://github.com/Automattic/wp-calypso/blob/9803c096ab0f779679e607cd81ea2ef114ea0b21/client/my-sites/post-type-list/post-actions-ellipsis-menu/duplicate.jsx#L57-L72

```js
return {
    ...,
    siteId = getSelectedSiteId( state ),
};
```